src/docs/flex-cli/manage-search-schemas-with-flex-cli/index.md

```diff
@@ -111,9 +120,19 @@ flex-cli search set --key category --type enum --scope public -m my-test-marketp
 flex-cli search set --key amenities --type multi-enum --scope public -m my-test-marketplace
 ```
 
+Note that we don't need to supply `--schema-for` option when we add schema for `listing` as `listing` is the default.
+
+## Adding user search schema
+
+We can also add a search schema for `userProfile` with `long` type.
+
+```shell
+flex-cli search set --schema-for userProfile --key age --type long --scope protected -m my-test-marketplace
+```
+
 We should now see the details of those schemas:
 
-![Search schemas for category and amenities](./search-schemas-added.png)
+![Search schemas for listing category, amenities and user age](./search-schemas-added.png)
```

Comment:
@wontheone1 the colors in the screenshot are a bit hard to separate - especially the bolded black ones. Can you adjust your terminal to have solid background and maybe different lighter color for the bolds? See, e.g. the old images - there the table column titles are visible better.

Reply:
BTW, you can see the resulting docs site by clicking the "Details" link on the Netlify automatic check that appears on the PR. The last one on the list " @netlify
netlify/sharetribe-flex-docs-site/deploy-preview — Deploy Preview ready! " is the one for the live preview.

That way you can more easily spot formatting issues and see the actual rendered articles.